gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderPage.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxNexusPageHeaderExample from './NxNexusPageHeaderExample';
+
+const nxNexusPageHeaderExampleCode = require('!!raw-loader!./NxNexusPageHeaderExample').default;
+
+const NxNexusPageHeaderPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        <code className="nx-code">NxNexusPageHeader</code> is a React component encapsulating the Sonatype Nexus
+        branded page header logo and structure. This header should only be used for products in the Sonatype Nexus
+        family.
+      </p>
+      <h3 className="nx-h3 nx-tile__subsection-header">Props</h3>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row">
+            <th className="nx-cell nx-cell--header">Prop</th>
+            <th className="nx-cell nx-cell--header">Type</th>
+            <th className="nx-cell nx-cell--header">Required</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
```

Comment:
Need docs for `imgURL`

Reply:
> Need docs for `imgURL`

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/2393c130c0d5c0ca17a958fbd02a7d6aae31c58e